scylla/Cargo.toml

```diff
@@ -57,6 +58,8 @@ rand_pcg = "0.3.1"
 socket2 = { version = "0.5.3", features = ["all"] }
 
 [dev-dependencies]
+num-bigint-03 = { package = "num-bigint", version = "0.3" }
+num-bigint-04 = { package = "num-bigint", version = "0.4" }
```

Comment:
Why do we need those? I thought it's for cql_types_test, but those previously used `num_bigint` so what changed?

Reply:
I just introduced test cases for both versions of the library. Previously we only had test cases for `num_bigint v0.3`.

Reply:
Previously there was no `num_bigint` dependency in dev-dependencies, now there are two. If the previous test cases didn't need `num_bigint` there why the new ones do?
Is it added so that in test those features are always enabled? If so, I don't think that's a good approach - we should be able to test with and without any feature, it should be controlled using command line options.

Reply:
> Previously there was no num_bigint dependency in dev-dependencies

You are right, but there was one in `[dependencies]` of scylla crate. The only things the dependency was used for were tests and `CqlValueDisplayer` (`scylla/src/utils/pretty.rs`).

Now, the dependency is not needed for `CqlValueDisplayer`, and so I moved it to `dev-dependencies`, so the tests can still use it (now we have two dependencies, one dependency per crate version).

> Is it added so that in test those features are always enabled?

Note that `dev-dependencies` cannot be specified as optional dependencies (and activated via feature flag). However, the test cases with specific `num-bigint` version are still enabled only when the corresponding feature flag is enabled, e.g.:
```rust
#[cfg(feature = "num-bigint-03")]
#[tokio::test]
async fn test_varint03() {
    let tests = varint_test_cases();
    run_tests::<num_bigint_03::BigInt>(&tests, "varint").await;
}
```

Reply:
Hmm... I can't say I love it, but no better solution comes to my mind right now. I think we can revisit if number of such dependencies grows too large.